src/picasso-def/components/not-fetched-pattern.js

```diff
@@ -1,29 +1,37 @@
-import { color as d3Color } from 'd3-color';
-
 const dummyCanvas = document.createElement('canvas');
 
 const ctx = dummyCanvas.getContext('2d');
 
-export function getPattern(color) {
-  const stroke = d3Color(color);
+export function getPattern() {
```

Comment:
maybe also update when getPattern is called (to not send any arguments)

Reply:
yes... !